readme/WindowsBuild.md

```diff
@@ -1,82 +1,78 @@
-# Xournal++ Windows Build
+# Windows Build
 
-![Screenshot](main-win.png?raw=true "Xournal++ Screenshot on Win10")
+![Screenshot](./main-win.png?raw=true "Xournal++ Screenshot on Win10")
+
+### Install Dependencies
```

Comment:
Any reason why this is level 3 instead of level 2?

Reply:
Nope - nice catch